### ISSUE TITLE
Page cache is too long. /read doesn't show recent posts until after "Updatinf post cache"

### ISSUE THREAD

**Issue by shleeable** [writefreely/writefreely#628]
### Describe the bug
New Blog does not invalidate the reader cache.

### Steps to reproduce (if necessary)
1) Post a new blog
2) load the /read page
3) blog is missing.
4) wait 10 minutes
5) reload the /readpage
6) blog is visible

### Expected behavior
Read should be updated on every post.

### Application configuration
 - **Single mode or Multi-user mode?**  Multi
 - **Database?** sqllite
 - **Open registration?** yes
 - **Federation enabled?** no

**Version or last commit**:
Latest tag

**Comment by shleeable** [comment]
The other side effect of this is the lag between deleting posts.. People can post something, delete it and it will be stuck in the reader cache for a while.

**Comment by thebaer** [comment]
Yep, this is definitely an issue that would be nice to fix. However, I think because the cache window is so small, this isn't super critical right now. But if anyone wants to address it, please feel free!

Also tracking this issue here: [T723](https://todo.musing.studio/T723).

**Comment by jbgi** [comment]
Also got this issue, I guess it's not critical if one know about the delay, but it can be very disturbing for users if not: "why my deleted post still in Reader, is it a bug?" "I very much don't want this post visible, need to contact the admin asap", etc.

**Comment by categulario** [comment]
I just faced this problem and found this issue in the 10 minute gap between when I published the post and when it appeared in the reader listing. It does create a feeling of "something is not working quite right". I had the knowledge to come here and find the issue but the less technical users of the platform might just find it frustrating.
